modules/sdk-core/src/bitgo/baseCoin/iBaseCoin.ts

```diff
@@ -503,6 +503,7 @@ export interface IBaseCoin {
   initiateRecovery(params: InitiateRecoveryOptions): never;
   parseTransaction(params: ParseTransactionOptions): Promise<ParsedTransaction>;
   generateKeyPair(seed?: Buffer): KeyPair;
+  generateRootKeyPair(seed?: Buffer): KeyPair;
```

Comment:
what's the difference between `generateKeyPair` and `generateRootKeyPair`? Isn't root key pair also just another keypair

Reply:
The format of the keys, eddsa coins return a coin specific format